### PR TITLE
Refactored ping and userinfo

### PR DESCRIPTION
**Pull request by SonDav**
Updated ping and userinfo from Moralebot to Moralebot2 with refactored v14 code. Removed return statements and awards section as requested. Both commands should work.

### CODE REVIEW

File: src/.env_template
Comment (michaelFarrier) [inline src/.env_template:1]: this file should not be deleted

**Review by michaelFarrier — CHANGES_REQUESTED** [review]

File: src/.env_template
Comment (SonDav) [reply src/.env_template:1]: FOR THE LAST TIME OLD MAN ITS NOT FKN DELETED IT'S RENAMED!

Comment (SonDav) [reply src/.env_template:1]: IT'S ALSO IN THE .GITIGNORE SO IT DOESN'T MATTER! READ!

Comment (SonDav) [reply src/.env_template:1]: Hey, just wanted to say... sorry about that buddy. 🥇 You dropped this king!

**Comment by SonDav** [comment]
We don't talk about this.

**Review by michaelFarrier — APPROVED** [review]
ok

**Comment by SonDav** [comment]
THAT'S RIGHT YOU IDIOT I DID EVERYTHING CORRECTLY!

**Pull request merged**